### ISSUE TITLE
Can't access /data/sacgf/microbio/lleong/commands/

### ISSUE THREAD

**Issue by tseemann** [lexleong/eyre#2]
You have helper scripts in `/data/sacgf/microbio/lleong/commands/`

These should be in the github repo?

**Comment by lexleong** [comment]
Yes. The helper scripts should be in this github site. I have modified the Snakefile, you might need to make those helper scripts executable.

**Comment by tseemann** [comment]
You'll need to add the helper scripts to github (don't forget to 'git push`)

**Comment by lexleong** [comment]
Hmm. I might be sounding very amature here (which I am! :P). But the helper script compile_seq_data.sh is in the lexleong/eyre dir
